controller/src/main/java/io/pravega/controller/store/PravegaTablesScope.java

```diff
@@ -121,42 +145,55 @@ public String getName() {
         Preconditions.checkNotNull(context, "Operation context cannot be null");
         return getStreamsInScopeTableName(true, context);
     }
-    
+
     public CompletableFuture<String> getStreamsInScopeTableName(boolean ignoreCached, OperationContext context) {
         Preconditions.checkNotNull(context, "Operation context cannot be null");
         if (ignoreCached) {
             storeHelper.invalidateCache(SCOPES_TABLE, scopeName);
         }
         return getId(context).thenApply(id ->
-                getQualifiedTableName(INTERNAL_SCOPE_NAME, scopeName, String.format(STREAMS_IN_SCOPE_TABLE_FORMAT, id.toString())));
+                                                getQualifiedTableName(INTERNAL_SCOPE_NAME, scopeName, String.format(STREAMS_IN_SCOPE_TABLE_FORMAT, id.toString())));
     }
 
     public CompletableFuture<String> getKVTablesInScopeTableName(OperationContext context) {
         Preconditions.checkNotNull(context, "Operation context cannot be null");
         return getKVTablesInScopeTableName(true, context);
     }
-    
+
     public CompletableFuture<String> getKVTablesInScopeTableName(boolean ignoreCached, OperationContext context) {
         Preconditions.checkNotNull(context, "Operation context cannot be null");
         if (ignoreCached) {
             storeHelper.invalidateCache(SCOPES_TABLE, scopeName);
         }
         return getId(context).thenApply(id ->
-                getQualifiedTableName(INTERNAL_SCOPE_NAME, scopeName, String.format(KVTABLES_IN_SCOPE_TABLE_FORMAT, id.toString())));
+                                                getQualifiedTableName(INTERNAL_SCOPE_NAME, scopeName, String.format(KVTABLES_IN_SCOPE_TABLE_FORMAT, id.toString())));
+    }
+
+    public CompletableFuture<String> getAllStreamTagsInScopeTableNames(String stream, OperationContext context) {
+        return getId(context)
+                .thenApply(id -> getQualifiedTableName(INTERNAL_SCOPE_NAME, scopeName,
+                                                       String.format(STREAM_TAGS_IN_SCOPE, id.toString(),
+                                                                     HASH.hashToBucket(stream, 25))));
```

Comment:
whats "25" here? should it be a constant?

Reply:
yes, thanks for pointing it. Fixed it.